CODE_OF_CONDUCT.md

```diff
@@ -62,6 +64,17 @@ affect a person's ability to participate within them.
   constructively. Remember that we are different. Different people
   have different perspectives on issues. Being unable to understand why someone
   holds a viewpoint doesn't mean that they're wrong.
+- **Be considerate of others' resources, including time and attention**
+  Long messages, for example, take more time for others to read and respond to, and
+  make it harder to determine what the most important points are,
+  risking miscommunication when multiple participants have different
+  views about what issue is being discussed. Short, concise messages are often more
+  efficient and help prevent communication misunderstandings.
+- **Avoid pressing for answers.**
+  The Coq community is generally highly responsive to users and
+  developers questions. However, in case you don't receive
+  an answer to your interrogation, there is no need to push for
+  response. 
```

Comment:
This seems factually incorrect, a question may be missed in which case asking again is the correct action

Reply:
What about clarifying that asking again once is OK, but what is not is to press for answers by repeatedly asking or pinging? Or do you think this is already covered by the point about harassment being when repeated actions that are OK individually become problematic as a pattern.

Reply:
>do you think this is already covered by the point about harassment being when repeated actions that are OK individually become problematic as a pattern.

I guess so.

Reply:
I'm fine with dropping this point, then.

Reply:
I have dropped this point.